### PR TITLE
Build: Fix prerelease script

### PR DESCRIPTION
**Pull request by nzakas**

### CODE REVIEW

File: Makefile.js [hunk 1]
```diff
@@ -182,64 +182,115 @@ function getReleaseType(version) {
...
+            releaseInfo["changelog_" + flag] = [];
+        }
+
+        releaseInfo["changelog_" + flag].push(log);
+    });
+
+    var output = logs.join("\n"); // and join it into a string
+    releaseInfo.raw = output;
+
+    if (releaseInfo.changelog_breaking) {
```
Comment (ilyavolodin) [inline Makefile.js]: What if at some point we wanted to release a major release that didn't have any breaking changes? As far as I understand, semversion allows major releases with no breaking change, it just doesn't allow minor releases with.

Comment (nzakas) [inline Makefile.js]: I can't think of a reason why we'd want to do that, but if such an occurrence happens, we could just put in a dummy commit that says "Breaking" and explains why we're doing that.

Comment (platinumazure) [inline Makefile.js]: Are you trying to remove the version type argument from the script invocation?

**Comment by ilyavolodin** [comment]
LGTM

**Pull request merged**